INSTALL

```diff
@@ -86,6 +86,9 @@ scipy 0.10 or later (http://www.scipy.org/)
 cartopy 0.11.0 or later (http://github.com/SciTools/cartopy/)
     Python package which provides cartographic tools for python.
 
+dask 0.11.0 or later (https://dask.pydata.org/)
```

Comment:
I have specified 0.11.0 because we use `dask.array.absolute` which was only included in 0.11.0

Note also the workaround for < and > dask 0.15 (see [here](https://github.com/SciTools/iris/blob/7ec22fa3f14b1aea454c5a51ac350de190603d9c/lib/iris/_lazy_data.py#L32))

I do find it concerning that we are using dask which hasn't released a major version (i.e. dask 1.0.0) yet